homeworks_js/h.w-26/javascript.js

```diff
@@ -0,0 +1,110 @@
+//1. Подсчитать сумму всех чисел в заданном пользователем диапазоне.
+let firstNumber = Number(prompt("Enter the start of the range",""));
+let lastNumber = Number(prompt("Enter the end of the range",""));
+let sum = 0;
+ 
+while(firstNumber <= lastNumber){       
+       sum+=firstNumber;  
+       firstNumber++;
+    }
+ 
+console.log(sum)
+
+//2. Запросить 2 числа и найти только наибольший общий делитель.
+let userFirstNumber = 86;
+let userSecondNumber = 90;
+
+while (userFirstNumber != 0 && userSecondNumber != 0) {
+    if (userFirstNumber > userSecondNumber) {
+        userFirstNumber = userFirstNumber % userSecondNumber }
+    else {
+        userSecondNumber = userSecondNumber % userFirstNumber
+    }
+}
+
+console.log(userFirstNumber + userSecondNumber)
+
+//3. Запросить у пользователя число и вывести все делители этого числа.
+let number = 21;
+for (let i = 1; i <= number; i++) {
+  if (number % i == 0) {
+    console.log(i);
+  }
+}
+
+//4. Определить количество цифр в введенном числе.
+let userNumber = Number(prompt("Enter your number"));
+while (number % 1 != 0) {}
+
+let g = 1;
+for (let i = 10; ; i *= 10) {
+    if (Math.abs(number) < i) {
+        console.log(`In your number there are ${g} numerics`);
+        break;
+    } else {
+        g++;
+    }
+}
+
+/*5. Запросить у пользователя 10 чисел и подсчитать, сколько он ввел положительных, отрицательных и нулей. При этом также посчитать, 
+сколько четных и нечетных. Вывести статистику на экран. Учтите, что достаточно одной переменной (не 10) для ввода чисел пользователем*/
+
+
+
+/*6. Зациклить калькулятор. Запросить у пользователя 2 числа и знак, решить пример, вывести результат и спросить, 
+хочет ли он решить еще один пример. И так до тех пор, пока пользователь не откажется.*/
+let firstNum;
+let secondNum;
+let operator;
+let result;
+let action;
+
+do {
+    firstNum = Number(prompt("Enter a first number"));
+    secondNum = Number(prompt("Enter a second number"));
+    operator = prompt("Enter an operator");
+
+    if (!firstNum || !secondNum || (operator != "+" && operator != "-" && operator != "*" && operator != "/")) {
```

Comment:
operator != "+" && operator != "-" && operator != "*" && operator != "/") this condition you can handle in  default stat,emt in switch